### DISCUSSION TITLE
disable hit area glow in osu!taiko argon pro skin

### DISCUSSION THREAD

**Discussion by Hiviexd** [ppy/osu#25087] [Ideas]
the aforementioned glow is quite distracting to play with on higher level maps, especially when at times it's constantly flashing. however i have to admit that it feels like a nice touch in terms of feedback for newer players, so i propose keeping this for the regular argon skin, and removing it for argon pro.

below is a comparison between argon pro and what every active taiko player is used to:

### argon pro

https://github.com/ppy/osu/assets/62819481/4f8143e8-591d-4889-b5e1-720bcade036e

### other skin

https://github.com/ppy/osu/assets/62819481/81b34634-2c09-4dc2-8e7d-a51e63657282

**Comment by peppy** [comment]
I'd be interested in toning it back before disabling it.
